example_softbody_gift: bounding box explosion (bbox_size > 10.0)

Sub-issue of #2313.

## Description

The `example_softbody_gift` example test fails with a bounding box explosion assertion:

```
AssertionError: Bounding box exploded: size=10.56
```

The assertion at [`example_softbody_gift.py:274`](https://github.com/newton-physics/newton/blob/main/newton/examples/multiphysics/example_softbody_gift.py#L274) checks `bbox_size < 10.0`, but the simulation produces values slightly above that threshold (~10.5–10.7).

## Reproduction

```bash
uv run -m newton.examples.multiphysics.example_softbody_gift --device cuda:0 --test --quiet --viewer null --num-frames 200
```

## Observed on

- **CI (nightly):** NVIDIA RTX PRO 6000 Blackwell (sm_120), `bbox_size=10.56` — see #2313 / [CI logs](https://github.com/newton-physics/newton/actions/runs/23976713844)
- **Local:** NVIDIA RTX 3090 (sm_86), `bbox_size=10.69`

Both environments use Warp 1.12.0 and CUDA Toolkit 12.9.

## Notes

- The non-manifold edge warnings during mesh construction may be contributing to instability.
- The failure is borderline (10.5–10.7 vs threshold of 10.0), suggesting either a slight simulation instability or an overly tight threshold.

Another occurrence in CI: https://github.com/newton-physics/newton/actions/runs/24383779810/job/71212984452

```
AssertionError: Bounding box exploded: size=10.07
```

Environment: g7e.2xlarge (L40S GPU), triggered by unrelated PR #2434 (Pillow upgrade).

--- a/newton/examples/multiphysics/example_softbody_gift.py
+++ b/newton/examples/multiphysics/example_softbody_gift.py
@@ -205,7 +205,7 @@ class Example:
         self.model = builder.finalize()
 
         # Contact parameters
-        self.model.soft_contact_ke = 1.0e5
+        self.model.soft_contact_ke = 5.0e4
         self.model.soft_contact_kd = 1e-5
         self.model.soft_contact_mu = 1.0
 

